docs/source/lambda.rst

```diff
@@ -62,10 +64,11 @@ Per Cluster Lambda Settings
 ===================  ========  ===========
 Key                  Required  Description
 -------------------  --------  -----------
-``timeout``          ``true``  The time (in seconds) the Lambda function is allowed to process an incoming record. The timeout can be set to any value between 1 and 300 seconds.
-``memory``           ``true``  The amount of memory allocated for the Lambda function execution.
-``current_version``  ``true``  The most current published version of the Lambda function.
-``outputs``          ``true``  A collection of S3 bucket IDs or AWS Lambda function names to configure as valid outputs.  By default, ``aws-s3`` should contain the bucket created by the ``stream_alert`` module: ``prefix.cluster.streamalerts``.  Optionally, if the alert processor needs to invoke other Lambda functions from within your AWS account, specify a list of function names.
+``timeout``          ``Yes``   The time (in seconds) the Lambda function is allowed to process an incoming record. The timeout can be set to any value between 1 and 300 seconds.
```

Comment:
@jacknagz  updated a few others spots in docs. we should get better about consistency when changes are first made instead of having to go back retroactively and update